mantidproject/mslice
# Cut using multiple workspaces

Can now select multiple  `_QE`  or  `_EQ` , then make a cut of each of them over the same range.
Can select multiple cuts that have been saved to workspace and display them at once.

**To test:**
For multiple workspaces:
- Load two files and calculate projections for each.
- Select the two `_QE`  or  `_EQ` workspaces, switch to cut tab, and input cut parameters.
- Click 'Plot'.

The same parameters should be used to make two cuts, one for each workspace, which are then displayed.

For multiple saved cuts:
- Load a file, calculate projections, input cut parameters.
- Click 'Save to Workspace'.
- Modify parameters and save more cuts to workspace.
- Select multiple cut workspaces and click 'Plot'.

All cuts are displayed at once.

Fixes #144
